runtime/Java/src/org/antlr/v4/runtime/ANTLRErrorStrategy.java

```diff
@@ -63,6 +63,9 @@
 	 * returns the {@link Token} instance which should be treated as the
 	 * successful result of the match.
 	 *
+	 * This method handles the consumption of any tokens - the caller should
```

Comment:
This paragraph should be wrapped in `<p>` and `</p>` tags so it displays properly when the documentation is generated.